unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -520,6 +523,18 @@ public void SetFacialFeaturesVisible(bool visible)
             bodyShapeController.SetFacialFeaturesVisible(visible, true);
         }
 
+        public void SetSSAOEnabled(bool newEnabled)
+        {
+            if (newEnabled == ssaoEnabled )
+                return;
+            ssaoEnabled = newEnabled;
+            if (bodyShapeController == null || !bodyShapeController.isReady)
+                return;
+            bodyShapeController.SetSSAOEnabled(ssaoEnabled);
+            foreach (WearableController wearableController in wearableControllers.Values)
+                wearableController.SetSSAOEnabled(ssaoEnabled);
```

Comment:
```suggestion
            foreach (WearableController wearableController in wearableControllers.Values)
            {
                wearableController.SetSSAOEnabled(ssaoEnabled);
            }
```
Small style suggestion